test/testdata/sandbox_config_seccomp.json

```diff
@@ -47,7 +47,6 @@
 	},
 	"annotations": {
 		"owner": "hmeng"
-		%VALUE%
```

Comment:
You can put it back, I guess.  This was blowing up when I tried to load this container using OCIC as badly formatted json.